d2bs/kolbot/libs/common/Config.js

```diff
@@ -436,7 +436,7 @@ var Config = {
 		RecheckSeals: false
 	},
 	MFHelper: {
-		BreakClearLevel: true
+		BreakClearLevel: false
```

Comment:
this is a breaking change, why the switch?

Reply:
https://d2bot.discourse.group/t/break-the-clearing/614

Reply:
to have the BreakClearLevel only in the case of executing MFHelper, which will stop the clearing in area if leader change that to other clearing area.

Reply:
why was BreakClearLevel even added? this should be handled differently altogether, not be a config option
https://github.com/blizzhackers/kolbot/pull/31/files

Reply:
anyone with a better idea could implement that in a new PR which will fix all issues related to the MFHelper